test/AutoRest.TestServer.Tests/TestMgmtParent.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using AutoRest.CSharp.AutoRest.Communication;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.AutoRest.Plugins;
+using AutoRest.CSharp.Input.Source;
+using AutoRest.CSharp.Output.Models.Types;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+
+namespace AutoRest.TestServer.Tests
+{
+    internal class TestMgmtParent
+    {
+        [Test]
+        public async Task TestParentComputer()
+        {
+            var basePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+            basePath = Path.Combine(basePath.Substring(0, basePath.IndexOf("autorest.csharp")), "autorest.csharp", "samples", "Azure.ResourceManager.Sample", "Generated");
```

Comment:
Can we directly create models required for these tests instead of loading a sample? Samples are based on real swagger that is not stable and can change in the future. At a minimum, we should use a hand-crafted test project swagger for this.